xml/cap_depl_service_broker_sidecar.xml

```diff
@@ -158,21 +158,21 @@
   
   # SCF cert
   CF_CA_CERT: |
-   -----BEGIN CERTIFICATE-----
-   MIIE8jCCAtqgAwIBAgIUT/Yu/Sv4UHl5zHZYZKCy5RKJqmYwDQYJKoZIhvcNAQEN
-   [...]
-   xC8x/+zT0QkvcRJBio5gg670+25KJQ==
-   -----END CERTIFICATE-----
-   
-   # UAA cert
-   UAA_CA_CERT: |
     -----BEGIN CERTIFICATE-----
-    MIIE8jCCAtqgAwIBAgIUSI02lj0a0InLb/zMrjNgW5d8EygwDQYJKoZIhvcNAQEN
+    MIIE8jCCAtqgAwIBAgIUT/Yu/Sv4UHl5zHZYZKCy5RKJqmYwDQYJKoZIhvcNAQEN
     [...]
-    to2GI8rPMb9W9fd2WwUXGEHTc+PqTg==
+    xC8x/+zT0QkvcRJBio5gg670+25KJQ==
     -----END CERTIFICATE-----
+   
+   # UAA cert
```

Comment:
The `# UAA cert/UAA_CA_CERT` section and its content are not aligned (1 space extra) with the entries above it.